content/tracing/languages/php.md

```diff
@@ -22,104 +22,254 @@ The APM tracer for PHP applications is in Open Public Beta.
+
+```php
+dd_trace("CustomDriver", "doWork", function (...$args) {
+    // Start a new span
+    $scope = GlobalTracer::get()->startActiveSpan('CustomDriver.doWork');
+    $span = $scope->getSpan();
+
+    // Access object members via $this
+    $span->setTag(Tags\RESOURCE_NAME, $this->workToDo);
+    
+    try {
+        // Execute the original method
+        $result = $this->doWork(...$args);
+        // Set a tag based on the return value
+        $span->setTag('doWork.size', count($result));
+        return $result;
+    } catch (Exception $e) {
+        // Inform the tracer that there was an exception thrown
+        $span->setError($e);
+        // Bubble up the exception
+        throw $e
+    } finally {
+        // Close the span
+        $span->finish();
+    }
+});
+```
+
+## Configuration
+
+Configure the agent connection parameters via environment variables.
+
+| Env variable               | Default     | Note                                                                       |
+| :------------------------- | :---------- | :------------------------------------------------------------------------- |
+| `DD_TRACE_ENABLED`         | `true`      | Enable the tracer globally                                                 |
+| `DD_INTEGRATIONS_DISABLED` | `null`      | CSV list of disabled extensions; e.g., `curl,mysqli`                       |
+| `DD_AGENT_HOST`            | `localhost` | The Agent host name                                                        |
+| `DD_TRACE_AGENT_PORT`      | `8126`      | The Agent port number                                                      |
+| `DD_AUTOFINISH_SPANS`      | `false`     | Whether or not spans are automatically finished when the tracer is flushed |
 
 ### Integrations
 
 #### Framework Compatibility
 
-| Module         | Versions    | Support Type       |
-| :-----------   | :---------- | :----------------- |
-| Laravel        | 5.x         | Public Beta        |
-| Laravel        | 4.2         | Coming Soon        |
-| Symfony        | 4.x         | Public Beta        |
-| Symfony        | 3.x         | Public Beta        |
-| Magento        | 2.x         | Coming Soon        |
-| Zend Framework | 3.x         | Coming Soon        |
-| CakePHP        | 3.x         | Coming Soon        |
-| Drupal         | 7.x         | Coming Soon        |
-| Wordpress      | 4.x         | Coming Soon        |
-| Yii            | 2.0.x       | Coming Soon        |
-| Slim           | 3.x         | Coming Soon        |
-
-Don't see your desired web frameworks? Let Datadog know more about your needs through [this survey][8]
+| Module         | Versions | Support Type |
+| :------------- | :------- | :----------- |
+| Laravel        | 5.x      | Beta         |
+| Laravel        | 4.2      | Beta         |
+| Symfony        | 4.x      | Beta         |
+| Symfony        | >= 3.3   | Beta         |
```

Comment:
\>= 3.4 :(

Reply:
Will do! I was thinking the timing of the integration would land around the time of this PR getting merged. I'm trying to predict the future too much! :D